### PR TITLE
Fix Missing .NET API documentation

### LINKED ISSUE #1993: API reference for Vector<T> missing
The API reference for [`System.​Numerics.Vector<​T>`](https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector-1?view=netcore-1.1) and [`System.​Numerics.Vector`](https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector?view=netcore-1.1) seems to be missing all content. The API is there, but there aren't any descriptions.

Few other, potentially relevant observations:

* Other `System.​Numerics` types (including e.g. [`Vector2`](https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector2?view=netcore-1.1)) seem to be fine.
* The types seem to be present only on .Net Core, even though they are actually supported on .Net Framework.
* When I select .Net Framework in the dropdown, a message shows saying:

   > Liquid error: Can't find the localized string disclaimer.moniker for template Reference.

* Google offers two different MSDN pages when searching for e.g. "vector t count":

   * https://msdn.microsoft.com/en-us/library/dn877911(v=vs.111).aspx, which doesn't have any site navigation (doesn't show up in TOC, does not have breadcrumbs). It links to https://msdn.microsoft.com/en-us/library/system.numerics(v=vs.111).aspx as its namespace.
   * https://msdn.microsoft.com/en-us/library/mt598437(v=vs.111).aspx, which seems to work fine. It links to https://msdn.microsoft.com/en-us/library/mt598257(v=vs.111).aspx as its namespace.

### PR DESCRIPTION
**Pull request by daxianji007**
Fix Missing .NET API documentation: https://mseng.visualstudio.com/VSChina/_workitems?id=979781

- dv_ng_codepages_mref
- dv_ng_dataflow_mref
- dv_ng_immutable_mref
- dv_ng_simd_mref

Fixes #1993

### CODE REVIEW

File: add/metadata/System.Text/DecoderFallback.meta.md [hunk 1]
```diff
@@ -1,41 +1,23 @@
 ---
 uid: System.Text.DecoderFallback
-author: "rpetrusha"
```
Comment (mairaw) [inline add/metadata/System.Text/DecoderFallback.meta.md]: why are these metadata values being removed?

**Review by mairaw — COMMENTED** [review]
Some metadata information seems to be removed and you need to sign the CLA please. Thanks!

**Comment by mairaw** [comment]
Thanks for signing the CLA @daxianji007. In addition to my previous question, we also have build errors. Is that something you can take care of?

**Comment by daxianji007** [comment]
@mairaw, I have updated the pull request:
1. re-migrate the meta.md
2. remove the empty token
This should resolve the existing problems.

File: xml/ns-System.Text.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Text">
   <Docs>
-    <summary>The <see cref="N:System.Text" /> namespace contains classes that represent ASCII and Unicode character encodings; abstract base classes for converting blocks of characters to and from blocks of bytes; and a helper class that manipulates and formats <see cref="T:System.String" /> objects without creating intermediate instances of <see cref="T:System.String" />.</summary>
+    <summary>The <see cref="N:System.Text" /> namespace in the System.Text.Encoding.CodePages.dll assembly contains a single class, <see cref="T:System.Text.CodePagesEncodingProvider" />, that extends the <see cref="T:System.Text.EncodingProvider" /> class to make code page encodings available to apps that target the Universal Windows Platform.</summary>
```
Comment (mairaw) [inline xml/ns-System.Text.xml]: @rpetrusha now that we'll only have a single namespace page for this, do you want to merge their content or leave the original one?

File: xml/ns-System.Numerics.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Numerics">
   <Docs>
-    <summary>The <see cref="N:System.Numerics" /> namespace contains numeric types that complement the numeric primitives, such as <see cref="T:System.Byte" />, <see cref="T:System.Double" />, and <see cref="T:System.Int32" />, that are defined by the .NET Framework.</summary>
+    <summary>Provides a set of SIMD-enabled vector types.</summary>
```
Comment (mairaw) [inline xml/ns-System.Numerics.xml]: should we leave the original here @rpetrusha?

**Comment by mairaw** [comment]
Thanks @daxianji007. There seems to be still some missing content that should have come with this migration.

For example, the explicit operators for `Vector<T>`: https://review.docs.microsoft.com/en-us/dotnet/api/system.numerics.vector-1.op_explicit?view=netcore-1.1&branch=pr-en-us-2046#System_Numerics_Vector_1_op_Explicit_System_Numerics_Vector_T__to_System_Numerics_Vector_System_UInt64__  

Their docs do exist in CAPS:
https://msdn.microsoft.com/en-us/library/dn858385(v=vs.111).aspx   

https://www.microsoftonedoc.com/#/organizations/e6f6a65cf14f462597b64ac058dbe1d0/projects/3fedad16-eaf1-41a6-8f96-0c1949c68f32/containers/5eb68fe7-2cae-4842-9f5b-686ef7ecd8a1/tocpaths/f296a9ed-a89d-4906-97d5-8c26d49f88f5/locales/en-US

File: xml/ns-System.Text.xml (hunk 2 above)
Comment (rpetrusha) [inline xml/ns-System.Text.xml]: @mairaw, Here I think that we want to leave the original one.

File: xml/ns-System.Numerics.xml (hunk 3 above)
Comment (rpetrusha) [inline xml/ns-System.Numerics.xml]: @mairaw, Yes, the original one. Except for one type not in the .NET Framework, the .NET Framework SIMD documentation is identical to the .NET Core documentation.

**Comment by daxianji007** [comment]
@mairaw , I fixed a bug that cause these missing Vector<T> contents and had a re-migration.

**Review by mairaw — APPROVED** [review]

**Pull request merged**